src/lib/plugin.js

```diff
@@ -308,11 +312,56 @@ class FiveBellsLedger extends EventEmitter2 {
     return res.body.balance
   }
 
-  send (transfer) {
-    return co.wrap(this._send).call(this, transfer)
+  /**
+   * @param {Object} message
+   * @param {IlpAddress} message.account
+   * @param {IlpAddress} message.ledger
+   * @param {Object} message.data
+   * @param {Object} message.custom (optional)
+   * @returns {Promise.<null>}
+   */
+  sendMessage (message) {
+    return co.wrap(this._sendMessage).call(this, message)
   }
 
-  * _send (transfer) {
+  * _sendMessage (message) {
+    if (message.ledger !== this.prefix) {
+      throw new errors.InvalidFieldsError('invalid ledger')
+    }
+    if (typeof message.account !== 'string') {
+      throw new errors.InvalidFieldsError('invalid account')
+    }
+    if (typeof message.data !== 'object') {
+      throw new errors.InvalidFieldsError('invalid data')
+    }
```

Comment:
No, `sendMessage` expects `message.ledger` to be a ledger prefix. `_handleMessageNotification` expects it to be the ledger host.